File: eTierMeeting_v2/SPA/src/app/_core/_models/eTM_Meeting_Log.ts
export interface eTM_Meeting_Log {
    record_ID: string;
    tU_ID: string;
    data_Date: string;
    meeting_Start_Time: Date;
    meeting_End_Time: Date | null;
    record_Status: string;
    duration_Sec: number | null;
    insert_By: string;
    insert_Time: Date;
    update_By: string;
    update_Time: Date | null;
}

File: eTierMeeting_v2/API/Models/eTM_Meeting_Log.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace eTierV2_API.Models
{
    public class eTM_Meeting_Log
    {
        [Key]
        public Guid Record_ID { get; set; }
        [Required]
        [StringLength(10)]
        public string TU_ID { get; set; }
        [Column(TypeName = "date")]
        public DateTime Data_Date { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime Meeting_Start_Time { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? Meeting_End_Time { get; set; }
        [Required]
        [StringLength(20)]
        public string Record_Status { get; set; }
        public int? Duration_Sec { get; set; }
        [Required]
        [StringLength(10)]
        public string Insert_By { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime Insert_Time { get; set; }
        [StringLength(10)]
        public string Update_By { get; set; }
        [Column(TypeName = "datetime")]
        public DateTime? Update_Time { get; set; }
    }
}